### PR TITLE
[html-to-pdfmake] Updates & definition extension

### PR DESCRIPTION
**Pull request by cwoebker**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [html-to-pdfmake repo README.md](https://github.com/Aymkdn/html-to-pdfmake/blob/master/README.md)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/html-to-pdfmake/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for html-to-pdfmake 2.1
+// Type definitions for html-to-pdfmake 2.4
 // Project: https://github.com/Aymkdn/html-to-pdfmake#readme
 // Definitions by: Martin Zloch <https://github.com/me>
+//                 Cecil Wöbker <https:/>/github.com/cwoebker>
```
Comment (MysteryBlokHed) [inline types/html-to-pdfmake/index.d.ts:4]:
```suggestion
//                 Cecil Wöbker <https://github.com/cwoebker>
```

Comment (cwoebker) [reply types/html-to-pdfmake/index.d.ts]: Thanks!

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by cwoebker** [comment]
Ready to merge

**Pull request merged**